src/goSuggest.ts

```diff
@@ -29,6 +30,32 @@ function vscodeKindFromGoCodeClass(kind: string): vscode.CompletionItemKind {
 	return vscode.CompletionItemKind.Property; // TODO@EG additional mappings needed?
 }
 
+function packageNameSuggestion(filename: string): Promise<string> {
+	return new Promise((resolve, reject) => {
+		const goFilename = basename(filename);
+		if (goFilename === 'main.go') {
+			return resolve('main');
+		}
+
+		if (goFilename.endsWith('internal_test.go')) {
+			return resolve(basename(dirname(filename)));
```

Comment:
Can you point me to any docs that talk about this as a standard?

Reply:
There are practice to make a test package to test un-exported method when necessary.

No official docs but here:
1. https://medium.com/@matryer/5-simple-tips-and-tricks-for-writing-unit-tests-in-golang-619653f90742
2. https://blog.alexellis.io/golang-writing-unit-tests/

Hmm.. I found not too much:
```
$ find /usr/local/Cellar/go/1.9/libexec/src | grep internal_test.go
/usr/local/Cellar/go/1.9/libexec/src/net/http/transport_internal_test.go
/usr/local/Cellar/go/1.9/libexec/src/os/exec/internal_test.go
/usr/local/Cellar/go/1.9/libexec/src/strconv/internal_test.go
/usr/local/Cellar/go/1.9/libexec/src/time/internal_test.go
```